### PR TITLE
Add guide register delegate, Add guide vote unvote, #41, #42

### PR DESCRIPTION
**Pull request by a1300**

### CODE REVIEW

**Comment by a1300** [comment]
@xpgeng please review this PR also after I merged it, I will fix issues later

Thanks 👍

File: docs/guide/get-gny-on-testnet.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<!-- address: GuPmGpiHfdnokkABdgimn21v24qj -->
+
+We will use the address `GuPmGpiHfdnokkABdg...` for our example.
+
+Paste **your** address into the input field in the `Step 2` section on [`faucet.gny.io`](faucet.gny.io):
+
+![faucet form 2](../.vuepress/public/faucet_paste_address.png)
+
+::: tip
+Check your `GNY token balance with the [check balance](./check-balance.md) guide
```
Comment (xpgeng) [inline docs/guide/get-gny-on-testnet.md]: Is here lack of another " ` "?

File: docs/guide/vote-for-delegate.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
+::: warning
+In order to be able to vote you need to first a part of your GNY tokens for a certain time period. Follow the [lock account for voting](./lock-account-for-voting.md) guide
```
Comment (xpgeng) [inline docs/guide/vote-for-delegate.md]: Shoud be `first lock a part of ...`

File: docs/guide/register-as-delegate.md [hunk 3]
```diff
@@ -1,13 +1,21 @@
...
-WIP
+```bash
+gny-cli --host "45.76.215.117" --port 4096 registerdelegate --secret ""
+```
+
+<br/>
 
-## With Wallet (GUI)
+::: tip
+After you successfully registered as `delegate` be sure to add your account secret to your forging node in order to enable forging. Forging will be possible if you are beeing voted in the top **101 delegates**
```
Comment (xpgeng) [inline docs/guide/register-as-delegate.md]: A typo：`beeing voted` -> `being voted`

**Pull request merged**